.github/workflows/chase_release_managers.yml

```diff
@@ -0,0 +1,34 @@
+name: "Chase missing release managers in confluence"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Full version of the release to schedule (e.g. 7.31.0)'
+        required: true
+        type: string
+
+
+jobs:
+  create_release_schedule:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Should work targeting the release tag
```suggestion
      uses: actions/checkout@v4.1.4
```

Reply:
It's a security recommendation, you can't change the commit whereas you could change where the version points. This works with dependabot (with version as comment)